packages/mobile/src/screens/search-screen-v2/components/SelectMoodScreen.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState } from 'react'
+
+import { View, Image } from 'react-native'
+
+import { IconMood, Text } from '@audius/harmony-native'
+import { ListSelectionScreen } from 'app/screens/edit-track-screen/screens/ListSelectionScreen'
+import { makeStyles } from 'app/styles'
+import { moodMap } from 'app/utils/moods'
+
+const messages = {
+  screenTitle: 'Mood',
+  searchText: 'Select Mood'
+}
+
+const moods = Object.keys(moodMap).map((mood) => ({
+  value: mood,
+  label: mood
+}))
+
+moods.sort((mood1, mood2) => mood1.label.localeCompare(mood2.label))
```

Comment:
Strange that we don't have them sorted already in the hardcoded list

Reply:
yea idk if there is a reason for this order that they usually come in